velox/exec/LocalPartition.h

```diff
@@ -120,6 +128,11 @@ class LocalExchangeSourceOperator : public SourceOperator {
 
   RowVectorPtr getOutput() override;
 
+  void close() override {
+    Operator::close();
```

Comment:
not sure if it matters in here, but conceptually, shouldn't we close the specialized resources before we move on to close the parent's resources (what if there's a dependency?)